src/pages/Cast.tsx

```diff
@@ -40,6 +45,12 @@ export default function () {
   const [text, setText] = useState('')
   useScrollToTop()
 
+  catchUnhandledRejection((error: unknown) => {
+    handleError(new Error(ErrorList.failedPost))
```

Comment:
`handleError` will `console.error` as well, isn't it?

Reply:
Yep, it has inside its own `console.error`. Removed `console` from  catchUnhandledRejection